open-sphere-plugins/open-street-map/src/test/java/io/opensphere/osm/server/OSMServerSourceControllerTest.java

```diff
@@ -39,6 +55,11 @@
      */
     private static final String ourServer = "http://osm.geointservices.io/osm_tiles_pc/{z}/{x}/{y}.png";
 
+    /**
+     * The url used in the server ping.
+     */
+    private static final String pingUrlString = "http://osm.geointservices.io/osm_tiles_pc/0/0/0.png";

```

Comment:
though it matches the other constant, use the all uppercase method: PING_URL_STRING.

Reply:
If you're feeling ambitious, fix the other one too.